docs/02-providers/aws/events/05-streams.md

```diff
@@ -0,0 +1,34 @@
+<!--
+title: DynamoDB / Kinesis Streams configuration docs
+menuText: DynamoDB / Kinesis Streams config
+layout: Doc
+-->
+
+# DynamoDB / Kinesis Streams
+
+This setup specifies that the `compute` function should be triggered whenever the corresponding DynamoDB table is modified (e.g. a new entry is added).
```

Comment:
Mention that we do not create the DynamoDB, but it only connects to a previously created one.